packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -79,7 +100,10 @@ const createProjectPrompt = async (
       message: i18n(`${i18nKey}.enterLocation`),
       when: !promptOptions.location,
       default: answers => {
-        return path.resolve(getCwd(), answers.name || promptOptions.name);
+        const projectName = sanitizeProjectName(
```

Comment:
Should we apply the sanitization to the validate method as well?  Like if `sanitizeProjectName(input) !== input` we display an error letting them know it's invalid?

Reply:
I would really love to validate the project's name, but we're trying to mimic the limits that the BE team has set on project names, and they have literally none. You can technically name a project just emojis 😭

Reply:
Isn't the `validate` method below for the `location` on the filesystem?

Reply:
+1 to Joe's question here. From what I can tell, this new location prompt will sanitize the initial default value that the CLI shows you, but it won't stop you from changing it to something that we don't want to allow. Does it make sense to move this over to the validation method for location?

Reply:
Ok, some clarification:

-Sorry, @joe-yeager, I misunderstood your initial ask, I thought you were talking about project name. So we can't simply check with `sanitizeProjectName(input) !== input`. The customer can put *anything* into the location field, we just supply the default value. We would need to perform two separate functions: one to sanitize the project name for the default location value, and one to validate the final location that the user provides.  
-To answer your second question, @brandenrodgers, we need to sanitize the project name to create the `default` location; validating the final location the user provides is a separate issue.